Assets/Scripts/Models/World.cs

```diff
@@ -632,7 +632,7 @@ void LoadInventoryPrototypesFromFile(string inventoryXmlText)
         //furniturePrototypes["Door"].RegisterUpdateAction( FurnitureActions.Door_UpdateAction );
         //furniturePrototypes["Door"].IsEnterable = FurnitureActions.Door_IsEnterable;
 
-            //Logger.LogError("Did not find a 'Inventories' element in the prototype definition file.");
+        //Logger.LogError("Did not find a 'Inventories' element in the prototype definition file.");
```

Comment:
I'm trying to figure out why it even says I changed that line... I guess at some point I changed the spacing... strange.